src/PureCloud.Client/Contracts/ILanguagesApi.cs

```diff
@@ -4,11 +4,43 @@ namespace PureCloud.Client.Contracts;
 
 public interface ILanguagesApi
 {
+    /// <summary>
+    /// Delete a language (Deprecated)
+    /// </summary>
```

Comment:
Document every parameter in every method. Apply the same feedback everywhere back everywhere